app/controllers/staff/omniauth_callbacks_controller.rb

```diff
@@ -0,0 +1,20 @@
+class Staff::OmniauthCallbacksController < Devise::OmniauthCallbacksController
+  def azure_activedirectory_v2
+    auth = request.env["omniauth.auth"]
+    email = auth["info"]["email"]
+    provider = auth["provider"]
+    uid = auth["uid"]
+
+    @staff = Staff.find_by(email:, provider: "azure_activedirectory_v2")
```

Comment:
```suggestion
    @staff = Staff.find_by(email:)
```

I think this would allow us to "migrate" the accounts over by getting assessors to sign in via ActiveDirectory and then it would link their account up using the email address - is this an approach we might want to take?